presets/workspace/tuning/text-generation/fine_tuning.py

```diff
@@ -124,7 +125,8 @@ def on_step_end(self, args, state: TrainerState, control: TrainerControl, **kwar
 ))
 trainer.train()
 os.makedirs(ta_args.output_dir, exist_ok=True)
-trainer.save_model(ta_args.output_dir)
+# only save the adapter weights
```

Comment:
Problem with this is that if the tuning crashes there is no way to resume from where it crashed from

Reply:
However I agree that saving an entire checkpoints is cumbersome. Can you still do checkpoint but do adapter-only checkpointing? @cvvz

Reply:
Actually, the check point wasn't used before, because it saved checkpoints and other model files in empty dir and only push the adapter to image registry, once the tuning pod completed or failed, the empty dir will be cleaned. 

If customer want to save the checkpoint in PV for resuming, they can  set `save_strategy` as "epoch", but now the behaviour is the same as before.